renderer/components/editor/video/index.js

```diff
@@ -0,0 +1,313 @@
+    <style jsx>{`
+    .current-time {
+      font-size: 12px;
+      line-height: 18px;
+      margin-top: -2px;
+      margin-left: 16px;
+      text-shadow: 0 1px 2px rgba(0,0,0,0.10);
+      font-weight: 100;
+    }
+  `}</style>
+  </div>
+);
+CurrentTime.propTypes = {
+  currentTime: PropTypes.number.isRequired
+};
+
+const getTimestampAtEvent = (event, duration) => {
+  const rect = event.target.getBoundingClientRect();
+  const xPositionInTrimmer = event.pageX - rect.left;
+
+  return duration * (xPositionInTrimmer / rect.width); // Calculated time in seconds where the click happened
+};
+
+const PlayBar = ({skip, startTime, duration = 0, previewDuration, currentTime = 0, scale = 1}) => {
+  const left = startTime * scale;
+  const roundedLeft = startTime === 0 ? '' : 'play-bar--rounded';
+  return (
+    <React.Fragment>
+      <div className="play-bar play-bar--background" style={{opacity: duration === previewDuration ? 0 : 1}}/>
+      <div className="play-bar play-bar--playable" style={{
+        width: `${(previewDuration * scale)}px`,
+        left: `${left}px`
+      }}/>
+      <div
+        className={`play-bar play-bar--current-time ${roundedLeft}`}
+        style={{
+          width: `${(currentTime * scale)}px`,
+          left: `${left}px`
+        }}
+      />
+      <div onClick={event => {
+        const time = getTimestampAtEvent(event, duration);
+        skip(time);
+      }} className="play-bar play-bar--clickarea"/>
+      <style jsx>{
+        `    
+        .play-bar {
+          position: absolute;
+          height: 4px;
+          left: 0;
+          bottom: 0;
+          border-radius: 0px 2px 2px 0px;
+          pointer-events: none;
+        }
+        .play-bar--clickarea {
+          pointer-events: auto;
+          -webkit-app-region: no-drag;
+          height: 30px;
+          transform: translateY(50%);
+          z-index: 100;
+          width: 100%;
+        }
+        .play-bar--current-time {
+          background: linear-gradient(90deg, #7146FE 0%, #00FFBE 100%);
+        }
+        .play-bar--background {
+          box-shadow: 0 1px 2px rgba(0,0,0,.1);
+          transition: opacity 200ms ease;
+          background: rgba(255,255,255,.40);
+          width: 100%;
+        }
+        .play-bar--playable {
+          background: rgba(255,255,255,.40);
+          width: 100%;
+        }
+        .play-bar--rounded {
+          border-radius: 2px 2px 2px 2px;
+        }
+        `
+      }</style>
+    </React.Fragment>
+  );
+};
+
+PlayBar.propTypes = {
+  skip: PropTypes.func.isRequired,
+  duration: PropTypes.number,
+  currentTime: PropTypes.number,
+  scale: PropTypes.number,
+  startTime: PropTypes.number,
+  previewDuration: PropTypes.number
+};
+
+export default class Video extends React.Component {
+  static propTypes = {
+    src: PropTypes.string.isRequired
+  }
+
+  state = {currentTime: 0, duration: null, endTime: null, startTime: 0, isPlaying: false, width: 0, hasFocus: false}
```

Comment:
Generally, if there are more than two prop, it's better to put them on their own lines.